### ISSUE TITLE
Questions about the quality metrics meaning

### ISSUE THREAD

**Issue by YatingPan** [Eyened/rtnls_vascx_models#1]
Hi, thanks for sharing this fantastic work! I am running the quality model on a local dataset, and I got the quality.csv where each image has 3 quality metrics as "q1", "q2", and "q3". What do these metrics mean, and what quality characteristics do they measure? I haven't found any clear guidance in the paper.

Thank you very much!

**Comment by denbonte** [comment]
Hey @YatingPan !

@josedvq correct me if I am wrong, but I think these are quality scores aligned with the [EyeQ scores](https://github.com/HzFu/EyeQ) (i.e., inferred by a network trained to label quality after the score in the EyeQ dataset). 

In short, `q1` should be for "reject", `q2` should be for "usable", and `q3` for "good". If you softmax the values in each of the q's column, they will sum up to one!

How you use the scores is up to you - e.g., you could define an aggregated quality score:

```
2*q3 + q2 # q1 is weighted 0 in this sum
```

Or decide to drop all the images for which q1 is the highest probability (although you might end up dropping half of your dataset if you go down this path, so be careful!)

**Comment by josedvq** [comment]
hi @YatingPan @denbonte  yes, that is exactly correct: they are logists for the quality bins used in EyeQ. They measure general image quality assessed subjectively by graders.
